src/game/main.cpp

```diff
@@ -269,6 +276,43 @@ void Create_Window()
 #endif
 }
 
+void Create_Window_SDL2()
+{
+#ifdef BUILD_WITH_SDL2
```

Comment:
How many SDL2 function do we expect to be created from here on? I think this is fine for now, but depending on the SDL2 volume in the future, we may want to not have every SDL2 function have this `#ifdef BUILD_WITH_SDL2`.

Reply:
I'd probably want to put these into different cpp files at some point and compile these conditionally. However since there isn't too much WinAPI code in Thyme yet i also can't replace that much